src/lib/bootloader/bootloader_factory.rb

```diff
@@ -108,6 +114,10 @@ def use_systemd_boot?
           (Yast::Arch.x86_64 || Yast::Arch.aarch64) # only these architectures are supported.
       end
 
+      def use_grub2_bls?
+        (Yast::Arch.x86_64 || Yast::Arch.aarch64) # only these architectures are supported.
```

Comment:
so no plan yet to try it on s390x? I read a bit that IBM people trying somehow BLS on s390

Reply:
The bootloader team is working on s390, but for now makes sense to make a step in the grub2-bls direction for intel and arm